### PR TITLE
Block NPM updates, rename node => npm

### PR DESCRIPTION
**Pull request by flashadvocate**
Certain projects may include a `package.json` file, but should not have their dependencies built in production because the source files are built locally and compiled assets are pushed to VCS

This PR also changes the `update_node` signature to `update_npm` since that's what we're actually doing

### CODE REVIEW

**Comment by flashadvocate** [comment]
@apschultz happy to invert this check if you want. But I figured this would be less invasive

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,2 @@
 .idea
-config.json
\ No newline at end of file
+deploy.config.json
```
Comment (apschultz) [inline .gitignore]: This shouldn't be needed. The actual config file is local to the bot, not the deploy git

File: manage.py [hunk 2]
```diff
@@ -34,8 +34,8 @@ def main(args):
         deploy_project(project_config)
     elif args.action == 'update-php':
         update_php_packages(project_config)
-    elif args.action == 'update-node':
-        update_node_packages(project_config)
+    elif args.action == 'update-npm':
```
Comment (apschultz) [inline manage.py:37]: Make sure to update the discord bot deploy.js

File: manage.py [hunk 3]
```diff
@@ -263,13 +263,17 @@ def update_php_packages(project_config):
         sys.exit(1)
 
 def update_node_packages(project_config):
+    if 'block_npm_updates' in project_config:
```
Comment (apschultz) [inline manage.py]: Should probably also check that the value is True.

File: .gitignore (hunk 1 above)
Comment (flashadvocate) [reply .gitignore]: We should probably drop the example file then and just keep the definition in the readme, since that implies the opposite

Comment (apschultz) [reply .gitignore]: I'm ok with that as long as we have a reasonable definition somewhere

**Review by apschultz — APPROVED** [review]

**Pull request merged**